### DISCUSSION TITLE
Opener

### DISCUSSION THREAD

**Discussion by Eastdoes** [atom/atom#22267] [Q&A]
How do I get to just to a text editing screen.

**Comment by ThatXliner** [comment]
<kbd>cmd</kbd>/<kbd>ctrl</kbd> + <kbd>n</kbd> for a new blank document

<kbd>cmd</kbd>/<kbd>ctrl</kbd> + <kbd>o</kbd> to open a file

See also: https://flight-manual.atom.io/getting-started/sections/atom-basics/#opening-modifying-and-saving-files, https://flight-manual.atom.io/using-atom/sections/editing-and-deleting-text/

**Reply by Eastdoes** [reply]
tysm
